src/Core/Services/AuthService.cs

```diff
@@ -610,8 +610,16 @@ public async Task<List<PasswordlessLoginResponse>> GetActivePasswordlessLoginReq
         }
         public async Task<PasswordlessLoginResponse> GetPasswordlessLoginRequestByIdAsync(string id)
         {
-            var response = await _apiService.GetAuthRequestAsync(id);
-            return await PopulateFingerprintPhraseAsync(response, await _stateService.GetEmailAsync());
+            try
+            {
+                var response = await _apiService.GetAuthRequestAsync(id);
+                return await PopulateFingerprintPhraseAsync(response, await _stateService.GetEmailAsync());
+            }
+            catch (ApiException ex) when (ex?.Error?.StatusCode == System.Net.HttpStatusCode.NotFound)
```

Comment:
⛏️ `ex` won't ever be `null` here:
```suggestion
            catch (ApiException ex) when (ex.Error?.StatusCode == System.Net.HttpStatusCode.NotFound)
```